Add proofs of some randomly selected problems

InnovativeInventor opened the pull request:
I got nerd-sniped and formalized some randomly selected problems from miniF2F to get an idea of the amount of work required. This is probably not representative; perhaps the first few I selected are too easy (did not encounter any real number ugliness, yet). 

I picked these by randomly scrolling through the other formalizations, ignoring the theorems that had "sorry". This probably biased the random selection towards the trivially solvable ones.

InnovativeInventor committed:
Add proofs of some randomly selected problems

spolu commented:
Extremely exciting to see the first Coq statements/proofs!

spolu commented:
Happy to merge as is. Would love to get someone else to give a thumbs up for good hygiene, just let me know 👍

InnovativeInventor committed:
Replace weaker translation with stronger, more accurate translation

I didn't realize that even though this is given as a multiple choice
problem, the other translations (e.g. in Lean) translate the sentence
as a forall (rather than an existential) sentence. This fixes the issue
and brings the translation in-line with the others.

InnovativeInventor committed:
Add two more proofs

InnovativeInventor committed:
Add two more proofs
